views/js/runner/plugins/security/fullScreen.js

```diff
@@ -124,7 +130,9 @@ define([
      */
     function checkFullScreen() {
         if (fullScreenProperty in doc) {
-            return !!doc[fullScreenProperty];
+            const genericFullScreen = !!doc[fullScreenProperty];
+            const mozWebkitFullScreen = screen.width === window.outerWidth && screen.height === window.outerHeight;
```

Comment:
**nitpick:** I wonder if the name is accurate. Is it actually related to Mozilla or Webkit? I'd rather name it `screenSizeCheck` or `isSizeFullscreen`...